mbed_flasher/flashers/FlasherPyOCD.py

```diff
@@ -218,21 +103,26 @@ def erase(self, target, no_reset):
 
         return EXIT_CODE_SUCCESS
 
-    def _get_session(self, target, error_class):
+    def _get_session(self, target, platform, pack, error_class):
         """
         Internal method for acquiring pyOCD session
         :param target: mbedls given target dictionary
+        :param platform: target platform
+        :param pack: path of pack file
         :param error_class: error to be risen on failure
         :return: Session
         """
+        assert isinstance(platform, str)
+        assert isinstance(pack, str) or pack is None
```

Comment:
Could we not use these asserts? They raise AssertionError and I don't think any methods or users handle this error currently. Also, if you add exceptions please document them in the docstring.

Reply:
They are in a "private" function and both public methods using them will translate the assert error to either FlashError or EraseError.

Reply:
This is a public project, there might be users we don't know about.

Reply:
And no, even the CLI does not handle this error.

Reply:
Oh wait, it's handled in a different location.

Reply:
Well, please at least add into the docstring, since that already exists and it documents params and the return value.